brave/brave-core
# fix(wallet): Make Buy and Deposit Screens Full Height

## Description 
Makes the `Buy` and `Deposit` screen full height.


Resolves <https://github.com/brave/brave-browser/issues/37437>
Resolves <https://github.com/brave/brave-browser/issues/37479>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open the `Wallet` and navigate to the `Buy` screen.
2. The `Select Token` sections should be the full height of the window.
3. Navigate to the `Deposit` screen
4. The `Select Token` sections should be the full height of the window.

https://github.com/brave/brave-core/assets/40611140/37621b08-0395-47d8-95ac-7fa3c71f4b4d
